### PR TITLE
Improved the getGraphEdge() function documentation

### PR DESCRIPTION
**Pull request by guidolodetti**

### CODE REVIEW

File: docs/FacebookResponse.fbmd [hunk 1]
```diff
@@ -181,7 +181,23 @@ Returns the response data in the form of a [`Facebook\GraphNodes\GraphUser`](/do
...
 ~~~~
-public Facebook\GraphNodes\GraphEdge getGraphEdge()
+public Facebook\GraphNodes\GraphEdge getGraphEdge(
+	string|null $subclassName,
+	boolean $auto_prefix)
 ~~~~
 Returns the response data in the form of a [`Facebook\GraphNodes\GraphEdge`](/docs/php/GraphEdge) collection.
+
+`$subclassName`
+The `Facebook\GraphNodes\GraphNode` sub class to cast list items to. If none is provided, default is `Facebook\GraphNodes\GraphNode`.
```
Comment (SammyK) [inline docs/FacebookResponse.fbmd]: "sub class" should be "subclass" :)

File: docs/FacebookResponse.fbmd [hunk 2]
```diff
@@ -181,7 +181,23 @@ Returns the response data in the form of a [`Facebook\GraphNodes\GraphUser`](/do
...
+
+`$auto_prefix`
+Toggle to auto-prefix the subclass name. If none is provided, default is `true`.
+
+~~~~
+$res = $fb->get('/{facebook-page}/events', '{access-token}');
+$events = $res->getGraphEdge("GraphEvent");
+foreach($events : $event){
```
Comment (SammyK) [inline docs/FacebookResponse.fbmd]: `foreach($events : $event){` should be `foreach ($events as $event) {` (with spacing too) :)

**Comment by guidolodetti** [comment]
Fixed it! Also, I've updated some fixed some *sub class* errors (that's why I've written "sub class")!

**Comment by SammyK** [comment]
:+1:

**Pull request merged**